notebooker/notebook_templates_example/local_context/hello_module.py

```diff
@@ -0,0 +1,8 @@
+# + {"tags": ["parameters"]}
+x_n = 2000
+y_n = 2070
+# -
```

Comment:
I think these tags/params can/should be removed.

How does Notebooker differentiate this from a jupytext-generated template so that it doesn't serve this as a potential notebook template in the frontend?